toolbox/datasets/oasst.py

```diff
@@ -0,0 +1,50 @@
+import json
+import os
+import typing as t
+
+from dataclasses import dataclass
+
+from toolbox.core.dataset import BaseDataset, get_path_for
+
+# Thanks GPT-4
+def get_list_of_dicts(d):
```

Comment:
This function is for exclusive use internal to this file, so I'd recommend renaming it to `_get_list_of_dicts` and moving it to the bottom of the file for consistency with the rest of the repo.